src/components/AmountInput/AmountInputTs.ts

```diff
@@ -101,16 +101,14 @@ export class AmountInputTs extends Vue {
     public get totalAvailableAmount() {
         const selectedMosaic = this.balanceMosaics.find((m) => m.mosaicIdHex === this.mosaicHex);
         const currentMosaicBalance = selectedMosaic.balance / Math.pow(10, selectedMosaic.divisibility);
-        const balance = this.isCosignatoryMode
-            ? currentMosaicBalance
-            : (selectedMosaic.balance - this.selectedFeeValue) / Math.pow(10, selectedMosaic.divisibility);
+        const balance =
+            this.isCosignatoryMode || selectedMosaic.mosaicIdHex !== this.networkConfiguration.currencyMosaicId
+                ? currentMosaicBalance
+                : (selectedMosaic.balance - this.selectedFeeValue) / Math.pow(10, selectedMosaic.divisibility);
         if (currentMosaicBalance == 0 || balance <= 0) {
```

Comment:
Looks like we can change this line to:
```
if (balance <= 0) {
```

could you verify?

Reply:
makes sense, update.